go/feast/basefeatureview.go

```diff
@@ -0,0 +1,43 @@
+package feast
```

Comment:
I think it would make sense to move the files under `go/feast/` into `go/internal`. Eventually we want to have them under a `pkg` folder so that people can import it, but I think it's reasonable to start with all of it under `internal`